oblt-cli/cluster-destroy/action.yml

```diff
@@ -22,7 +26,6 @@ runs:
     - uses: elastic/oblt-actions/oblt-cli/setup@v1.2.0
       with:
         github-token: ${{ inputs.github-token }}
-        slack-channel: ${{ inputs.slack-channel }}
```

Comment:
@kuisathaverat this caused the error because the input doesn't exist on the destroy action